common/supplemental/plurals.xml

```diff
@@ -225,6 +225,13 @@ CLDR data files are interpreted according to the LDML specification (http://unic
             <pluralRule count="many">v != 0   @decimal 0.0~1.5, 10.0, 100.0, 1000.0, 10000.0, 100000.0, 1000000.0, …</pluralRule>
             <pluralRule count="other"> @integer 3~10, 13~19, 23, 103, 1003, …</pluralRule>
         </pluralRules>
+        <pluralRules locales="sgs">
```

Comment:
Most languages don't require so many variations. Often it is not necessary to have this full plural explosion. However it may totally be a feature of Samogitian so its totally fine to go ahead with this change.

Can you show examples of why you'd have 5 plural classes? Linguistically it looks like Samogitian should only have 3: Singular (one), Dual (two), and Plural (other). Can you write out some minimal pairs to show the difference? [Example from Lithuanian](https://www.unicode.org/cldr/charts/45/supplemental/language_plural_rules.html#lt):
1 obuolys
2 obuoliai
0,5 obuolio
10 obuolių

Reply:
Yes, it is similar to Lithuanian, except Samogitian has dual form:
1 vuobols
2 vuobolo
3 vuobolē
0,5 vuobolė
10 vuoboliū

Lithuanian has Singular and Plural, but some quantities wants different inflection: vuobols, vuobolo, vuobolē (similar in Lithuanian obuolys, obuoliai) - all are in nominative and vuobolė (lt. obuolio) is singular genetive, vuololiū (lt. obuolių) is plural genetive